Contest_API.md

```diff
@@ -1460,6 +1464,7 @@ The request must fail with a 4xx HTTP status code if any of the following happen
 * A PATCH on an award id that doesn't exist.
 * A PATCH that contains an invalid attribute (e.g. null `citation` or `team_ids`).
 * A DELETE on an award id that doesn't exist.
+* A POST, PATCH, or DELETE on an award id that the server is configured to manage exclusively.
```

Comment:
Maybe add `the server or another client is configured to manage exclusively.`

Reply:
No, because it says *must* above.